processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/jump_hazard.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/jump_hazard.asm (1 of 1)
processing file asmFiles/test.rtype.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-05 18:12:01.295895309 -0400
+++ memcpu.hex	2021-10-05 18:12:01.231892887 -0400
@@ -1,5 +1,5 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
-:0400020027BD000412
 :040003008FA30000C7
 :0400040027BD000410
 :0400050034040001BE
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 18:12:04.819028613 -0400
+++ memcpu.hex	2021-10-05 18:12:04.748025926 -0400
@@ -1,6 +1,5 @@
 :04000000341DFFFCB0
 :040001003402001FA6
-:040002003403000AB9
 :0400030034040010B1
 :0400040027BDFFFC19
 :04000500AFA20000A6
@@ -31,6 +30,6 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.ldst2.asm (2 of 2)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 18:47:17.225971130 -0400
+++ memcpu.hex	2021-10-05 18:47:17.091966058 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/jump_hazard.asm (1 of 1)
processing file asmFiles/jal.asm (1 of 1)
processing file asmFiles/jal.asm (1 of 2)
processing file asmFiles/jump_hazard.asm (2 of 2)
processing file asmFiles/test_halt.asm (1 of 3)
processing file asmFiles/test_jal_jr.asm (2 of 3)
processing file asmFiles/test_j.asm (3 of 3)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/RAW.asm (1 of 1)
--- memsim.hex	2021-10-05 19:01:48.544948336 -0400
+++ memcpu.hex	2021-10-05 19:01:48.451944816 -0400
@@ -1,4 +1,3 @@
-:04000000340100F0D7
 :0400010034020001C4
 :0400020034030001C2
 :0400030034050002BE
@@ -6,5 +5,4 @@
 :0400050000A3302004
 :04000600AC2300081F
 :04000700FFFFFFFFF9
-:04003E00000011139A
 :00000001FF
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
--- memsim.hex	2021-10-05 19:09:00.591299143 -0400
+++ memcpu.hex	2021-10-05 19:09:00.477294828 -0400
@@ -13,6 +13,7 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040020000000BA5CC6
+:040021000000BA5CC5
+:0400240000000018C0
 :00000001FF
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
--- memsim.hex	2021-10-05 19:09:07.219549989 -0400
+++ memcpu.hex	2021-10-05 19:09:07.083544842 -0400
@@ -11,6 +11,5 @@
 :04000A00340400506A
 :04000B00ACA300089A
 :04000C00FFFFFFFFF4
-:043FC20000000005F6
 :043FC30000000020DA
 :00000001FF
processing file asmFiles/test_j.asm (4 of 8)
--- memsim.hex	2021-10-05 19:09:10.760684003 -0400
+++ memcpu.hex	2021-10-05 19:09:10.654679992 -0400
@@ -8,5 +8,5 @@
 :0400070034020009B6
 :04000800AC820008BE
 :04000900FFFFFFFFF7
-:043FC20000000009F2
+:043FC20000000020DB
 :00000001FF
processing file asmFiles/test.ldst1.asm (5 of 8)
--- memsim.hex	2021-10-05 19:09:14.245815898 -0400
+++ memcpu.hex	2021-10-05 19:09:14.103810524 -0400
@@ -15,7 +15,7 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:04004000BAD1BAD1A6
+:04004100BAD1BAD1A5
+:04004200BAD1BAD1A4
 :00000001FF
processing file asmFiles/test.ldst2.asm (6 of 8)
--- memsim.hex	2021-10-05 19:09:17.681945939 -0400
+++ memcpu.hex	2021-10-05 19:09:17.561941397 -0400
@@ -10,9 +10,9 @@
 :04000A00AC450008F9
 :04000B00AC4A000CEF
 :04000C00FFFFFFFFF4
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
+:04020000BAD1BAD1E4
+:04020100BAD1BAD1E3
+:04020200BAD1BAD1E2
 :04020300FEEDBEEF5F
 :0403C000000073378F
 :0403C1000000270110
processing file asmFiles/test.loadstore.asm (7 of 8)
--- memsim.hex	2021-10-05 19:09:20.967070264 -0400
+++ memcpu.hex	2021-10-05 19:09:20.868066518 -0400
@@ -10,9 +10,9 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
+:04002000BAD1BAD1C6
+:04002100BAD1BAD1C5
+:04002200BAD1BAD1C4
 :04002300DEADBEEFA1
 :04003C000000733716
 :04003D000000270197
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/branch_hazard.asm (1 of 1)
--- memsim.hex	2021-10-05 19:09:33.711552580 -0400
+++ memcpu.hex	2021-10-05 19:09:33.571547281 -0400
@@ -7,5 +7,5 @@
 :04000800FFFFFFFFF8
 :04000900AC25000022
 :04000A00FFFFFFFFF6
-:04003C0000000002BE
+:04003C0000000001BF
 :00000001FF
processing file asmFiles/jal.asm (1 of 2)
--- memsim.hex	2021-10-05 19:09:40.976827534 -0400
+++ memcpu.hex	2021-10-05 19:09:40.832822084 -0400
@@ -10,5 +10,5 @@
 :04000900340400506B
 :04000A00ACA300089B
 :04000B00FFFFFFFFF5
-:043FC20000000009F2
+:043FC30000000020DA
 :00000001FF
processing file asmFiles/jump_hazard.asm (2 of 2)
--- memsim.hex	2021-10-05 19:09:45.457997124 -0400
+++ memcpu.hex	2021-10-05 19:09:45.330992317 -0400
@@ -7,5 +7,6 @@
 :04000600FFFFFFFFFA
 :04000700AC2400081D
 :04000800FFFFFFFFF8
+:04003D0000000001BE
 :04003E0000000002BC
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
--- memsim.hex	2021-10-05 19:09:54.050322301 -0400
+++ memcpu.hex	2021-10-05 19:09:53.910317002 -0400
@@ -14,7 +14,7 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:04002000000000904C
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2021-10-05 19:10:00.940583063 -0400
+++ memcpu.hex	2021-10-05 19:10:00.793577499 -0400
@@ -12,27 +12,6 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400220075A375A2AB
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-05 19:10:07.572834061 -0400
+++ memcpu.hex	2021-10-05 19:10:07.450829443 -0400
@@ -1,3 +1,4 @@
+:04000000BAD1BAD1E6
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 19:10:10.753954450 -0400
+++ memcpu.hex	2021-10-05 19:10:10.641950211 -0400
@@ -31,6 +31,6 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 19:11:28.851910070 -0400
+++ memcpu.hex	2021-10-05 19:11:28.711904771 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/branch_hazard.asm (1 of 1)
--- memsim.hex	2021-10-05 19:34:09.275388490 -0400
+++ memcpu.hex	2021-10-05 19:22:32.916038822 -0400
@@ -7,5 +7,5 @@
 :04000600FFFFFFFFFA
 :04000700AC25000024
 :04000800FFFFFFFFF8
-:04003C0000000002BE
+:04003C0000000001BF
 :00000001FF
processing file asmFiles/branch_hazard.asm (1 of 1)
--- memsim.hex	2021-10-05 19:34:22.469887795 -0400
+++ memcpu.hex	2021-10-05 19:22:32.916038822 -0400
@@ -7,5 +7,5 @@
 :04000600FFFFFFFFFA
 :04000700AC25000024
 :04000800FFFFFFFFF8
-:04003C0000000002BE
+:04003C0000000001BF
 :00000001FF
processing file asmFiles/branch_hazard.asm (1 of 1)
--- memsim.hex	2021-10-05 19:35:08.763639637 -0400
+++ memcpu.hex	2021-10-05 19:35:08.702637329 -0400
@@ -7,5 +7,4 @@
 :04000600FFFFFFFFFA
 :04000700AC25000024
 :04000800FFFFFFFFF8
-:04003C0000000002BE
 :00000001FF
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
--- memsim.hex	2021-10-05 19:49:18.373790475 -0400
+++ memcpu.hex	2021-10-05 19:49:18.285787145 -0400
@@ -15,7 +15,7 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:04004000BAD1BAD1A6
+:04004100BAD1BAD1A5
+:04004200BAD1BAD1A4
 :00000001FF
processing file asmFiles/test.ldst2.asm (6 of 8)
--- memsim.hex	2021-10-05 19:49:22.013928225 -0400
+++ memcpu.hex	2021-10-05 19:49:21.913924440 -0400
@@ -10,9 +10,9 @@
 :04000A00AC450008F9
 :04000B00AC4A000CEF
 :04000C00FFFFFFFFF4
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
+:04020000BAD1BAD1E4
+:04020100BAD1BAD1E3
+:04020200BAD1BAD1E2
 :04020300FEEDBEEF5F
 :0403C000000073378F
 :0403C1000000270110
processing file asmFiles/test.loadstore.asm (7 of 8)
--- memsim.hex	2021-10-05 19:49:26.613102266 -0400
+++ memcpu.hex	2021-10-05 19:49:26.479097195 -0400
@@ -10,9 +10,9 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
+:04002000BAD1BAD1C6
+:04002100BAD1BAD1C5
+:04002200BAD1BAD1C4
 :04002300DEADBEEFA1
 :04003C000000733716
 :04003D000000270197
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2021-10-05 19:49:39.731598694 -0400
+++ memcpu.hex	2021-10-05 19:49:39.628594797 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
@@ -17,4 +18,5 @@
 :0400110027BDFFFC0C
 :04001200AFA6000095
 :04001300FFFFFFFFED
+:043FFF008B2CBD4208
 :00000001FF
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 19:49:44.220768573 -0400
+++ memcpu.hex	2021-10-05 19:49:44.081763313 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFC008B2CBD420B
+:043FFD000000000AB6
+:043FFE000000001FA0
 :00000001FF
processing file asmFiles/load_use.asm (1 of 1)
--- memsim.hex	2021-10-05 19:50:02.671466872 -0400
+++ memcpu.hex	2021-10-05 19:50:02.551462330 -0400
@@ -7,5 +7,6 @@
 :04000600AC23000423
 :040007002063111150
 :04000800FFFFFFFFF8
-:04003C00000011119E
+:04003C00BAD1CBE288
+:04003D00BAD1BAD1A9
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 19:51:32.245857065 -0400
+++ memcpu.hex	2021-10-05 19:51:32.137852977 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
